my_list.rb

```diff
@@ -0,0 +1,28 @@
+require_relative './my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] Please, make use of the yield keyword to invoke a block of code into all members of @list in succession.